test/client/end-to-end.clienttest.js

```diff
@@ -10,9 +9,9 @@ module.exports = {
       .assert.urlEquals('http://localhost:8000/search?q=ada')
       .click('.resultcard')
       .pause(1000)
-      .assert.urlEquals('http://localhost:8000/people/cp38764/ada-lovelace')
+      .assert.urlEquals('http://localhost:8000/objects/co8228143/fort-ada-alexandria-photograph')
```

Comment:
Yeah, I think you're right that the test should be checking that a result we'd expect to be the top result is, and if not, we'd need to tweak the search parameters. I don't think we should do this until after the upgrade to es5 though, as the changes to the search will likely be outdated.